drivers/pwm/Kconfig.ifx_cat1

```diff
@@ -0,0 +1,15 @@
+# Infineon CAT1 PWM configuration options
+
+# Copyright (c) 2023 Cypress Semiconductor Corporation (an Infineon company) or
+# an affiliate of Cypress Semiconductor Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config PWM_INFINEON_CAT1
+	bool "Infineon CAT1 PWM driver"
+	default y
+	depends on DT_HAS_INFINEON_CAT1_PWM_ENABLED
+	select USE_INFINEON_PWM
```

Comment:
if it support cyw920829 now, we need to add `depends on SOC_FAMILY_INFINEON_CAT1B`

Reply:
Good point.  Done